pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -102,7 +102,10 @@ angular: teacherDashboard
       //  'remove': {method:'DELETE'},
       //  'delete': {method:'DELETE'}
          update: {method:'POST', url: 'v2/sections/:id/update'},
+         // TODO: This isn't very restful
+         allStudents: {method:'GET', url:'v2/sections/:id/students', isArray: true},
          addStudents: {method:'POST', url:'/v2/sections/:id/students', isArray: true},
+         moveStudents: {method:'POST', url:'/sections/:new_section_code/transfers'},
```

Comment:
It's not available in js so you have to use haml.. The js code is in a haml file so you should be able to generate it there, maybe using #{}

> On Jan 9, 2016, at 17:34, David Wu <notifications@github.com> wrote:
> 
> In pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml:
> 
> >           addStudents: {method:'POST', url:'/v2/sections/:id/students', isArray: true},
> > +         moveStudents: {method:'POST', url:'/sections/:new_section_code/transfers'},
> Thanks! Is there a way to the equivalent of CDO.studio_url in JS? The usages I've seen for it are in haml code, but when I try url: CDO.studio_url("sections/:new_section_code/transfers"), CDO is undefined.
> 
> —
> Reply to this email directly or view it on GitHub.
>